src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -76,6 +77,20 @@ const SubscriberHistory = ({
           campaignsPerPage,
           currentPage,
         );
+
+        const tabsProperties = [
+          {
+            url: `/reports/subscriber-history?email=${subscriber.email}`,
+            descriptionId: 'subscriber_history.title',
+            active: true,
+          },
+          {
+            url: `/reports/subscriber-gdpr?email=${subscriber.email}`,
+            descriptionId: 'subscriber_gdpr.page_title',
+            active: false,
```

Comment:
Another interesting thing, not related to the previous discussion, is to use a more idiomatic React code. For example, in place of using this kind of array, React developers do something like this:

```html
<TabGroup>
  <TabItem active url={`/reports/subscriber-history?email=${subscriber.email}`}>
    <FormattedMessage id="subscriber_history.title" />
  </TabItem>
  <TabItem url={`/reports/subscriber-gdpr?email=${subscriber.email}`}>
    <FormattedMessage id="subscriber_gdpr.page_title" />
  </TabItem>
</TabGroup>
```

Or maybe:

```html
<TabGroup currentUrl={currentUrl}>
  <TabItem url={`/reports/subscriber-history?email=${subscriber.email}`}>
    <FormattedMessage id="subscriber_history.title" />
  </TabItem>
  <TabItem url={`/reports/subscriber-gdpr?email=${subscriber.email}`}>
    <FormattedMessage id="subscriber_gdpr.page_title" />
  </TabItem>
</TabGroup>
```

Reply:
Interesting 😄 👍